internal/detector/jar_assessor.go

```diff
@@ -104,19 +102,16 @@ func (ja *JarAssessor) Assess(path string) (JarAssessement, error) {
 				semver, err := ParseSemver(v)
 				if err != nil {
 					logrus.Warnf("unable to parse semver: %s", err)
-					continue
 				}
-				version = semver
-				break
+				jarAssessment := JarAssessement{
```

Comment:
nitpick: could be `jarAssessment := &JarAssessement{` instead to avoid referencing it twice after

Reply:
done